api_app/analyzers_manager/migrations/0124_analyzer_config_androguard.py

```diff
@@ -0,0 +1,124 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+    ReverseManyToOneDescriptor,
+    ReverseOneToOneDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "androguard.AndroguardAnalyzer",
+        "base_path": "api_app.analyzers_manager.file_analyzers",
+    },
+    "name": "Androguard",
+    "description": "[Androguard]\r\n(https://github.com/androguard/androguard) is a full Python tool to reverse Android Applications.",
+    "disabled": False,
+    "soft_time_limit": 60,
+    "routing_key": "default",
+    "health_check_status": True,
+    "type": "file",
+    "docker_based": False,
+    "maximum_tlp": "AMBER",
+    "observable_supported": [],
+    "supported_filetypes": ["application/vnd.android.package-archive"],
```

Comment:
you can also add the following ones:
application/x-dex
application/zip
application/java-archive

That's because sometimes they get flagged like that. Also other APK tools use this list